src/app/(main)/agency/[agencyId]/all-subaccounts/page.tsx

```
@@ -0,0 +1,120 @@
+      <div className="flex flex-col ">
+        <CreateSubaccountButton
+          user={user}
+          id={params.agencyId}
+          className="w-[200px] self-end m-6"
+        />
+        <Command className="rounded-lg bg-transparent">
+          <CommandInput placeholder="Search Account..." />
+          <CommandList>
+            <CommandEmpty>No Results Found.</CommandEmpty>
+            <CommandGroup heading="Sub Accounts">
+              {!!user.Agency?.SubAccount.length ? (
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)